packages/cmf/README.md

```diff
@@ -254,8 +259,8 @@ you may change the following using simple props:
 
 For 1.0
 
-- [x] embedable apps
-- [ ] react-router v4
-- [ ] i18n
-- [x] generator
-- [x] actionCreator should become first class
+* [x] embedable apps
+* [ ] react-router v4
+* [ ] i18n
+* [x] generator
+* [x] actionCreator should become first class
```

Comment:
could you move all theses changes on #959 ?

Reply:
done